src/avfoundation.cs

```diff
@@ -1077,15 +1354,19 @@ interface AVAudioMixerNode : AVAudioMixing {
 	interface AVAudioOutputNode {
 
 	}	
+	
+	// AudioBufferList is binded as AudioBuffers which is present in AudioToolbox which is not present on the watch. TODO.
+	[Watch (4,0), TV (11,0), Mac (10,10), iOS (8,0)]
+	delegate IntPtr AVAudioIONodeInputBlock (uint frameCount);
 
-	[NoTV]
-	[NoWatch]
-	[iOS (8,0)][Mac (10,10)]
-	[BaseType (typeof (AVAudioIONode))]
-	[DisableDefaultCtor] // returned Handle is nil
-	// note: sample source (header) suggest it comes from AVAudioEngine properties
+ 	[Watch (4,0)]
+ 	[iOS (8,0)][Mac (10,10)][TV (11,0)]
+ 	[BaseType (typeof (AVAudioIONode))]
```

Comment:
It looks like you have unneeded whitespace changes here, because this line didn't actually change.